src/jukebox/components/controls/bluetooth_audio_buttons/README.rst

```diff
@@ -28,7 +28,7 @@ Custom key bindings
 You may change or extend the actions assigned to a button in the configuration. If the configuration contains
```

Comment:
General: I think we should make this a markdown file for consistency and it should be in the documentation folder

Reply:
Agree. I didn't yet change it for time aspects, as there are quite some changes needed.